tests/run/test_genericclass.py

```diff
@@ -162,7 +162,8 @@ class TestClassGetitem(unittest.TestCase):
     # BEGIN - Additional tests from cython
     def test_no_class_getitem(self):
         class C: ...
-        with self.assertRaises(TypeError):
+        # note that PyPy raises AttributeError on __class_getitem__
+        with self.assertRaises(TypeError if not hasattr(sys, "pypy_version_info") else AttributeError):
```

Comment:
@mattip I've worked around it, but I'd think `TypeError` is the right error to raise

Reply:
This is fixed on PyPy HEAD, thanks